src/module_PI/Raspberry_PI/main/PacketMessageTranslator.java

```diff
@@ -0,0 +1,110 @@
+package module_PI.Raspberry_PI.main;
+
+import java.util.*;
+
+public class PacketMessageTranslator {
+	
+	private Map<Integer, String> messageTable = new Hashtable<>();
+	private final int MAX_PACKET_DETAILS_INT = 55999;
+	private final int NUM_OF_BYTES_IN_PACKET_DETAILS = 3;
+	private final int NUM_OF_BYTES_IN_PACKET = 5;
+	private final int MAX_DETAILS_ONLY_INT = 999;
+	private final int MIN_FULL_PACKET_INT = 11001;
+	
+	public PacketMessageTranslator(){
+		buildTable();
+	}
+	
+	private void buildTable(){
+		messageTable.put(1, "Execute Command: Arm");
```

Comment:
What I read in that book was that you run tests with the expected data being hard coded. I figured that if we made it a variable, especially a variable we pull from the class, that it's not really an effective test. 

In regards to the ENUM, I thought about doing a similar data structure, but my hesitation is that the big-O would be N (or I guess N/2, which is still N) instead of 1 for every call, because it would need to loop through each entry to see if it matches our request. This would be slower, but more RAM friendly. I chose the RAM heavy instant one because the Arduino is going to be shooting messages at it pretty quickly. And I was excited to be able to use a HashTable.